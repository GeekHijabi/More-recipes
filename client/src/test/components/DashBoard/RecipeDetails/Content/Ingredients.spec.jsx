import React from 'react';
import { shallow } from 'enzyme';
import { Ingredients }
  from '../../../../../js/components/DashBoard/RecipeDetails/Content/Ingredients';

const props = {
  ingredients: 'water, rice',
  activeTab: {
    name: 'ingredients',
  }
};

describe('Ingredients snapshot', () => {
  it('should render correctly', () => {
    const ingredients = shallow(<Ingredients {...props} />);
    expect(ingredients).toMatchSnapshot();
  });
});
