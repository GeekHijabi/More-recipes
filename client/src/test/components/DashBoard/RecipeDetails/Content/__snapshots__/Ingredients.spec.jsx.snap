// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ingredients snapshot should render correctly 1`] = `
<div
  aria-labelledby="ingredients-tab"
  className="tab-pane fade"
  id="ingredients"
  role="tabpanel"
>
  <div
    className="card"
  >
    <div
      className="card-header"
    >
      Ingredients
    </div>
    <ul
      className="list-group list-group-flush"
    >
      <li
        className="list-group-item"
        key="ingredient 1"
      >
        water
      </li>
      <li
        className="list-group-item"
        key="ingredient 2"
      >
         rice
      </li>
    </ul>
  </div>
</div>
`;
